src/styles/blocks/card.scss

```
@@ -0,0 +1,112 @@
+@use '../utils/_variables' as *;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  box-sizing: border-box;
+  width: $card-width;
+  height: $card-height;
+  border: 1px solid $card-border-color;
+  border-radius: $card-border-radius;
+
+  background-color: $card-background-color;
+
+  &__info {
+    display: flex;
+    flex-direction: column;
+
+    box-sizing: border-box;
+    width: 100%;
+    padding: 0 17px;
```

you can move this padding to card class and apply it to the top and bottom as well instead of using separate margins 
```suggestion
    padding: 0 16px;
```